pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -352,3 +396,11 @@ func renderCrioConfigSnippet(profile *performancev2.PerformanceProfile, src stri
 
 	return crioConfig.Bytes(), nil
 }
+
+func listToString(cpus []int) string {
+	items := make([]string, len(cpus))
+	for idx, cpu := range cpus {
+		items[idx] = strconv.FormatInt(int64(cpu), 10)
+	}
+	return strings.Join(items, ",")
+}
```

Comment:
If you keep the function then please move it to one of the util folder such as `pkg/performanceprofile/utils` or `pkg/util`